hermes-mock/src/test/groovy/pl/allegro/tech/hermes/mock/TestMessage.groovy

```diff
@@ -0,0 +1,31 @@
+package pl.allegro.tech.hermes.mock
+
+import com.fasterxml.jackson.core.JsonProcessingException
+import com.fasterxml.jackson.databind.ObjectMapper
+
+class TestMessage {
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    String key
+    String value
+
+    TestMessage() {}
+
+    TestMessage(String key, Object value) {
+        this.key = key
+        this.value = value
+    }
+
+    static TestMessage random() {
+        return new TestMessage("random", UUID.randomUUID().toString());
+    }
+
+    @Override
+    String toString() {
+        try {
+            return objectMapper.writeValueAsString(this);
```

Comment:
In Groovy you can use `@ToString` annotation to have a nice `toString` generated at compile time: http://docs.groovy-lang.org/2.4.9/html/gapi/groovy/transform/ToString.html.